Tools/Pipeline/Common/PipelineController.cs

```diff
@@ -481,6 +481,12 @@ private string FindMGCB()
 
         private void DoBuild(string commands)
         {
+            if(!Environment.Is64BitOperatingSystem)
+            {
+                View.OutputAppend("Build failed:  Content can only be compiled from a 64 bit OS.");
```

Comment:
Sorry... I missed your point again.

You're saying that since `<ForceArchitecture>x64</ForceArchitecture>` is defined then the MGCB.exe and 2MGFX.exe have to be 64bit and the check of `Environment.Is64BitOperatingSystem` is pointless.

We have had people report issues in the past where mojoshader or some other native 64bit binary would not load for them.... suggesting they launched MGCB.exe or 2MGFX.exe successfully.  It could be that these are all old reports and we had been building still as "Any CPU", but i'm not sure of that.

Gonna do a test here on a 32bit tablet I have here.